utilities/verify.sh

```diff
@@ -10,7 +10,7 @@ if [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
     chmod 700 $TRAVIS_BUILD_DIR/signing-tools
     tar xvf $TRAVIS_BUILD_DIR/signing-tools.tar -C $TRAVIS_BUILD_DIR/signing-tools
     # Run verify
-    mvn verify
+    mvn verify -P release
```

Comment:
Any reason for this change? I mean what happens if I don't specify "-P release"?
At the verify stage I think it is enough to generate the javadoc (so we can see any javadoc related errors) however
creating a jar for them is not that important (and no need for a source jar).